_pytest/logging.py

```diff
@@ -2,15 +2,73 @@
 
 import logging
 from contextlib import closing, contextmanager
+import re
 import six
 
 import pytest
 import py
 
 
+LOGLEVEL_COLOROPTS = {
+    logging.CRITICAL: {'red'},
+    logging.ERROR: {'red', 'bold'},
+    logging.WARNING: {'yellow'},
+    logging.WARN: {'yellow'},
+    logging.INFO: {'green'},
+    logging.DEBUG: {'purple'},
+    logging.NOTSET: set(),
+}
+
 DEFAULT_LOG_FORMAT = '%(filename)-25s %(lineno)4d %(levelname)-8s %(message)s'
 DEFAULT_LOG_DATE_FORMAT = '%H:%M:%S'
 
+# move to __init__ of ColoredLevelFormatter?
+LEVELNAME_FMT_REGEX = re.compile(r'%\(levelname\)([+-]?\d*s)')
```

Comment:
Probably both `LEVELNAME_FMT_REGEX` and `LOGLEVEL_COLOROPTS` would be better off being class attributes of `ColoredLevelFormatter`.